### ISSUE TITLE
Minor Bug Bundle

### ISSUE THREAD

**Issue by GuiTaek** [Ladysnake/Requiem#659]
Some time ago, I have a lot of small bugs, that don't hurt very much that I thought I would bundle into one Bug Report. I tested this on the latest  version 2.0.0-beta.16. Some bugs seemed to be already fixed (marked with a tick):

- [ ] you loose golden apple while curing in creative
- [ ] being teleported to the nether through a nether portal as non-player or ghost instantly is understandable but weird in survival
- [ ] mortitians can't have their soul stolen with shift-use
- [ ] pillager attack possessed villagers in creative
- [ ] When you are a piglin (healed or not healed, not a piglin brute) and are damaged, you loose your offhand
- [ ] The rift teleport gui has in-code a possible image for selecting a rift, but it doesn't apply, because only 2 images are provided
- [x] no sound for mortictian trade
- [x] sometimes you hold items as a ghost
- [x] using the command tp with a non-player-hull will kill the entity
- [x] possessed pillager have the screen wobble when regenerating

**Comment by GuiTaek** [comment]
Wow, you were fast!

**Comment by Pyrofab** [comment]
We have a webhook on discord, sometimes I read it lol

Anyway, what's the nether teleportation thing you are referring to ?

**Comment by GuiTaek** [comment]
Does the update help?

**Comment by Pyrofab** [comment]
Ah yes okay. And regarding the selected rift code, doesn't the icon change on hover ? Or are you talking about something else ?

**Comment by GuiTaek** [comment]
Yes, the icon doesn't change on hover, there isn't even a part of the image designated for that ready implemented feature :) Anything I can add to the original bug report so it becomes clearer for the person wanting to fix this bug (if there is such a person 😂)?
